doc/api/vm.markdown

```diff
@@ -36,6 +36,12 @@ The options when creating a script are:
   code are controlled by the options to the script's methods.
 - `timeout`: a number of milliseconds to execute `code` before terminating
   execution. If execution is terminated, an [`Error`][] will be thrown.
+- `cachedData`: an optional `Buffer` with V8's code cache data for the supplied
+  source. When supplied `cachedDataRejected` value will be set to either
+  `true` or `false` depending on acceptance of the data by V8.
```

Comment:
What are the factors that determine whether or not V8 will accept the cached data or not? 

I'm trying to better understand it from reading the test cases, but will V8 only accept the cached data if the script compiled is the same?